### DISCUSSION TITLE
LazyVim Project Donations

### DISCUSSION THREAD

**Discussion by helgardferreira** [LazyVim/LazyVim#2786] [Q&A] [answered]
Hi, I'm not entirely sure what's the best way to reach out to @folke or what the best way to go about this would be - but I've been using LazyVim for my neovim setup (lazy.nvim for even longer) and since I've stopped using essentially any other text editing setup I'd like to make a recurring donation to the LazyVim project.

What would be the best way to go about doing this? Apologies if this post does not fit the conventional Q&A format.

**Comment by Raagh** [comment]
I am also interested in this

**Comment by chrisgeo** [comment]
Was going to make a bug report to get his attention and find out. Maybe it's a labor of love for @folke?

**Comment by folke** [comment, marked as answer]
Hi everyone, I really appreciate the gesture, but I'm in a position where I really don't need any financial support. I work on opensource projects just for fun.

If you'd like to support something, there's a ton of opensource projects out there, that might benefit from funding. Or you can always donate to a charity to make a difference.

Thanks again.

**Reply by akthe-at** [reply]
@folke ...I think you already had everyone's appreciation and respect...but damn that response takes it to another level. 👏

**Reply by Andras-Csanyi** [reply]
Cool, then more goes to NeoVim.
